modules/packages/Socket.chpl

```diff
@@ -567,10 +824,12 @@ const backlogDefault:uint(16) = (if SOMAXCONN <= 128 then SOMAXCONN else 128):ui
 
   :arg address: address to connect to
   :type address: :type:`ipAddr`
-  :arg timeout: standard ipv6 address.
-  :type timeval: :type:`~Sys.timeval`
+  :arg reuseAddr: whether to reuse address if already in use
+  :type reuseAddr: `bool`
+  :arg backlog: maximum number of pending connections
+  :type backlog: `uint(16)`
   :return: connected socket
-  :rtype: `tcpConn`
+  :rtype: `tcpListener`
```

Comment:
I wasn't grokking this diff at first, but it looks like you just found a case where the documentation was wrong/incomplete?  Thanks for fixing!